vm/chef/cookbooks/sonarqube/files/sonar-config-setup

```diff
@@ -47,32 +72,45 @@ $sonar_pass
 $sonar_pass
 EOF
 
-# Enabling Sonarqube.
+# Hide Sonarqube from end-users while default credentials are still set
+sed 's/sonar.web.host=0.0.0.0/sonar.web.host=127.0.0.1/g' -i "${sonarqube_properties}"
+
+# Enable Sonarqube.
 systemctl start sonar
 systemctl enable sonar
 
-# Await the application to be ready.
-function wait_for_app() {
-  local max_retries=10
-  local retry=1
-  local http_code=0
+# Wait frontend return 200
+wait_for_app "http://localhost" 200 ""
 
-  while [[ "${http_code}" -ne 200 && "${retry}" -le "${max_retries}" ]]; do
-    echo "Trying to fetch homepage... ${retry}"
-    http_code="$(curl -s -L -o /dev/null -w "%{http_code}" http://localhost)"
-    (( retry=retry+1 ))
-    sleep 25
-  done
+# Ensure admin/admin password.
+sudo -u sonar psql <<EOF
+  update users set
+  crypted_password='100000\$t2h8AtNs1AlCHuLobDjHQTn9XppwTIx88UjqUm4s8RsfTuXQHSd/fpFexAnewwPsO6jGFQUv/24DnO55hY6Xew==',
```

Comment:
should this be set to ${salted_password}?

otherwise, it seems like salted_password is not being used anywhere in this file?

Reply:
Actually no. There's no official reference on how to crypt those passwords so I'm updating the credentials via API. 
👍🏻  for the vars not used. Just removed it.